VelocidiSDK/VelocidiSDK/VSDKVelocidi.h

```diff
@@ -7,6 +7,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+  VelocidiSDK main class responsible for orchestrating all the logic.
```

Comment:
Just for the sake of consistency.

```suggestion
  VelocidiSDK main class responsible for orchestrating all the logic
```